amberdata/src/adapter.ts

```diff
@@ -21,6 +21,9 @@ export const execute: ExecuteWithConfig<Config> = async (request, config) => {
     case price.Name: {
```

Comment:
I can't comment on the line above, but we should convert the endpoint to lowercase before comparing, just so we avoid issues if the endpoint is defined as "marketCap".